dll/win32/riched20/para.c

```diff
@@ -312,7 +309,11 @@ static ME_String *para_num_get_str( ME_Paragraph *para, WORD num )
     {
     case PFN_ARABIC:
     default:
-        p += swprintf( p, fmtW, num );
+#ifdef __REACTOS__
+        p += swprintf( p, L"%d", num );
+#else
+        p += swprintf( p, 20, L"%d", num );
```

Comment:
I wonder whether you could directly use wine's version of the code here, if you add in riched20's cmake files the line:
`remove_definitions(-D_CRT_NON_CONFORMING_SWPRINTFS)`

Reply:
This does work. Should we make this change? Thanks.

Reply:
Changed as noted above. Thanks.